utils/rpms/daos.spec

```diff
@@ -247,7 +247,7 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %config(noreplace) %{conf_dir}/daos_server.yml
 %{_sysconfdir}/ld.so.conf.d/daos.conf
 # set daos_admin to be setuid root in order to perform privileged tasks
-%attr(4750,root,daos_admins) %{_bindir}/daos_admin
+%attr(4755,root,daos_admins) %{_bindir}/daos_admin
```

Comment:
This is incorrect. The daos_admin binary was deliberately made non-executable by other. It should only be executable by root or daos_admins.